src/ontology/cl-edit.owl

```diff
@@ -24511,7 +24524,7 @@ EquivalentClasses(obo:CL_1000143 ObjectIntersectionOf(obo:CL_0000160 ObjectSomeV
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25311230"^^xsd:string) obo:IAO_0000115 obo:CL_1000147 "A cell that is part of a cardiac valve."^^xsd:string)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30796046"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_1000147 "heart valve cell"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_1000147 "cardiac valve cell"^^xsd:string)
-EquivalentClasses(obo:CL_1000147 ObjectIntersectionOf(obo:CL_0000000 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000946)))
+EquivalentClasses(obo:CL_1000147 ObjectIntersectionOf(obo:CL_0000003 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000946)))
```

Comment:
See [this comment](https://github.com/obophenotype/cell-ontology/pull/1758#issuecomment-1339672528).

Reply:
hmmm this is odd given you didn't change this. If it's not your change, would you mind reverting this please? 
Thanks

Reply:
To clarify: I did make this change as noted in the linked comment.
The declared PATO, UBERON and PR are the parts I did not add.

Reply:
wait so do you want it to be native cell or cell?

Reply:
> The declared PATO, UBERON and PR are the parts I did not add.

This is strange, I always thought protege doesn't declare imports, not quite sure why this is all coming up. I vaguely rmbr there being some minor concerns with declaring imported terms (I think it has something to do with if they are declared but not used), but its not a HUGE issue. Let me check with other PRs and all and get back to you on this

Reply:
'native cell'

Reply:
yeah I'm not sure why protege is declaring all these terms, almost certain we don't want them. Are you using the new beta release? 
Either way could you try removing them please? (would do it but running a import on another branch atm)